### PR TITLE
Content tweaks to Decision Reviews/Appeals content

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR updates does the following -

- Removes the banner from https://www.va.gov/decision-reviews/ 
- Deletes the code for the banner that use to live at https://www.va.gov/disability/file-an-appeal/
   - This actual banner on the website lives in Drupal, so this is just a cleanup.
- Updates the content in the alert boxes of the File My Claim Online wizard

Ticket - https://github.com/department-of-veterans-affairs/va.gov-team/issues/5541
(See attached Word doc)

## Testing done
Visited each content change

### Banner gone from /decision-reviews
image

### File My Claim Online wizard - 
Visible on /disability/how-to-file-claim/

#### Date before Feb. 19, 2019
image

#### After Feb. 19, 2019
image

## Acceptance criteria
- [ ] Content is updated in various places

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/page-breadcrumbs.html [hunk 1]
```diff
@@ -4,14 +4,6 @@
   {% include "src/site/includes/gibs-down.html" %}
 {% endif %}
 
-{% if appealsbanner %}
-  {% include "src/site/includes/appeals-banner.html" %}
-{% endif %}
-
-{% if decision_reviews_banner %}
-  {% include "src/site/includes/decision-reviews-banner.html" %}
-{% endif %}
```
Comment (ncksllvn) [inline src/site/layouts/page-breadcrumbs.html]: image

File: src/applications/disability-benefits/wizard/pages/disagreeing.jsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ const DisagreeingPage = ({ setPageState, state = defaultState }) => {
     );
   return (
     <ErrorableDate
-      label="What’s the official date of VA’s decision?"
+      label="What’s the date of VA’s decision?"
```
Comment (ncksllvn) [inline src/applications/disability-benefits/wizard/pages/disagreeing.jsx]: image

File: src/applications/disability-benefits/wizard/pages/file-appeal.jsx [hunk 3]
```diff
@@ -5,20 +5,23 @@ import { pageNames } from './pageList';
...
+      </a>
     </p>
   </>
 );
 
 const FileAppealPage = () => (
   <AlertBox
     status="warning"
-    headline="You’ll need to file an appeal"
+    headline="You’ll need to file a Supplemental Claim"
```
Comment (ncksllvn) [inline src/applications/disability-benefits/wizard/pages/file-appeal.jsx]: image

File: src/applications/disability-benefits/wizard/pages/decision-review.jsx [hunk 4]
```diff
@@ -5,9 +5,14 @@ import { pageNames } from './pageList';
...
-      decision review on all or some of the decision. You have 1 year from the
-      date on your decision to request a review.
+      In most situations, you have one year from the date on your decision
+      letter to request a decision review. The deadline to file may be different
+      if you have a fiduciary claim, a{' '}
+      <a href="/decision-reviews/multiple-party-claims/">
+        claim contested by multiple parties
+      </a>
+      , or if you’re filing a{' '}
+      <a href="/decision-reviews/#supplemental-claim">Supplemental Claim</a>.
```
Comment (ncksllvn) [inline src/applications/disability-benefits/wizard/pages/decision-review.jsx]: image

**Review by kelsonic — APPROVED** [review]
Yay! Deletions > Additions! 🎉

**Review by bkjohnson — APPROVED** [review]
This seems fine from a platform perspective.

**Pull request merged**